policy/release/olm_test.rego

```diff
@@ -0,0 +1,76 @@
+package policy.release.olm
```

Comment:
See https://github.com/containerbuildsystem/operator-manifest/blob/master/tests/test_operator.py for the use cases that must be supported.